login_KA.feature

```diff
@@ -0,0 +1,144 @@
+    Given user has a gmail account with attached phone number to it
+    When user clicks "<change the account> button"
+    And fills phone number in the field
+    And clicks "<next> button"
+    And user fills invalid password in the "<password> field"
+    Then user sees error message and "password field" is marked red
+    
+    Examples:
+    | phone        |  | password       |
+    | +79123456789 |  | wrong_password |
+
+Scenario: 1_4.1 Log in using another account, which is not google one
+    Given user has yandex account
+    When user clicks "<change the account> button"
+    And user fills yndex login
+    And clicks "<next> button"
+    Then user seer the error message "google account is not finded" and "login field" marked red
+
+    Examples:
+    | login               |  
+    | viburnumk@yandex.ru |  
+
+Scenario: 1_5.1 Log in with empty login
+    Given user has a google account
+    When user clicks "<change the account> button"
+    And clicks "<next> button"
+    Then user seer the error message "enter email address or phone number" and "login field" marked red
+
+Scenario: 1_5.2 Log in with empty password
+    Given user has a google account
+    When user clicks "<change the account> button"
+    And fills valid login
+    And clicks "<next> button"
+    Then user seer the error message "enter password" and "password field" marked red
+
+Scenario: 1_6.1 Log in with fresh login
+   Given user does not have a google account
+   When user clicks "<change the account> button"
+   And clicks "<create an account> button"
+   And fills all required fields (name, family name, user name, password, prove password)
+   And clicks "<next> button"
+   And fills valid phone number
+   And enter code, which system sent to the phone
+   And fills reqired fields (birth date, gender)
+   And clicks "<next> button"
+   And clicks "<skip> button"
+   And clicks "fast personolization"
+   And clicks "<next> button"
+   And clicks "<prove> button"
+   And clicks "<accept> button"
+   Then user is logged in 
+   And user sees a main page
+
+    Examples:
+    | name |  | family name |  | user name       |  | password |  | prove password |  | birth date |  | gender |
+    | anna |  | kalinina    |  | annakalininarus |  | password |  | password       |  | 10.04.1995 |  | female |
+
+   "well I actually did this, I understand that there are negative tests here as well, but I do not want to create a bunch of gmail accounts so I would like to skip them"
+
+Scenario: 1_7.1 User log in with 2f authorisation on
+    Given user has a google account
+    When user clicks "<change the account> button"
+    And fills the valid login
+    And clicks "<next> button"
+    And fills the valid password
+    And clicks "<next>"
+    And passes two-factor authentication
+    Then user is logged in
+    And sees the main page
+
+    Examples:
+    | email                          |  | password       |
+    | anna.kalinina@quantoria.cademy |  | right_password |
```

Comment:
Understandable titles for keyword Scenario:
Basically, it is good written, correct  syntax and right order of keywords, understandable titles for keyword scenario
Enough covering for testing functionality at the point different login variants .
As for me, I would add scenarious with some actions in scoring to be sure that signed user can use app functionality 
e. Correct formatting (spaces, commas etc)